pkg/transparentproxy/config/config_executables.go

```diff
@@ -301,7 +227,22 @@ func (c InitializedExecutablesIPvX) Restore(
 	ctx context.Context,
 	rules string,
 ) (string, error) {
-	return c.restore(ctx, rules, FlagNoFlush)
+	f, err := createTempFile(c.IptablesRestore.prefix)
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	defer os.Remove(f.Name())
+
+	// Log the file name and the rules to be written.
+	c.logger.Info("writing the following rules to file:", f.Name())
```

Comment:
can't we have structured logs like we do in the rest of the project?

Reply:
This logger is very simple and I don't think it's worth to get another dependency, especially that we are running this code as part of kumactl. It's also important in CNI case to have full control of where the logs goes (stderr/stdout).

Reply:
> especially that we are running this code as part of kumactl

I mean... if we run this as a part of kumactl then we already have structure logger.

> It's also important in CNI case to have full control of where the logs goes (stderr/stdout)

Ok, that might be good enough reason. Is it because we put iptables on stdout and logs on stderr or why exactly?

Reply:
> I mean... if we run this as a part of kumactl then we already have structure logger
If I would include our structured logger it would add to this code dependency on kubernetes packages which would make it harder to move, and there are plans to move transparent proxy logic to kuma-dp

> Ok, that might be good enough reason. Is it because we put iptables on stdout and logs on stderr or why exactly?
To be honest I don't remember exactly, but kuma-cni is a binary executed by kubelet with some opinionated expectations on the stdout/stderr it receives.

Reply:
> and there are plans to move transparent proxy logic to kuma-dp

which also has a dependency on the same structure logger like any other part of the code. I don't want to block this PR over it, but IMHO there is a value of consistent logging around the project, so just think about it.